### ISSUE TITLE
 SummarizedExperiment object to matrix object?

### ISSUE THREAD

**Issue by sunarella** [rstudio/DT#440]
Hello,

Is there any way I could convert a Summarized Experiment object into a usual matrix format?
I am seeing the layout (samples on columns and genes on rows) that I want with datatable function, but I can only view without being able to manipulate..

count_data
class: RangedSummarizedExperiment
dim: 57035 529
metadata(0):
assays(1): HTSeq - Counts
rownames(57035): ENSG00000000003 ENSG00000000005 ... ENSG00000281912 ENSG00000281920
rowData names(3): ensembl_gene_id external_gene_name original_ensembl_gene_id
colnames(529): TCGA-S9-A6WP-01A-12R-A34F-07 TCGA-TM-A84T-01A-11R-A36H-07 ...
TCGA-HT-8109-01A-11R-2404-07 TCGA-W9-A837-01A-11R-A36H-07
colData names(106): patient barcode ...
subtype_Telomere.length.estimate.in.blood.normal..Kb.
subtype_Telomere.length.estimate.in.tumor..Kb.

So basically I want to convert this into a matrix (some format like xls. or txt.) of genes on the rows and TCGA sample barcodes on the columns.

Need your help, please!

**Comment by yihui** [comment]
Sorry, but I can only handle Github issues for bug reports and feature requests, and I don't have time to answer general questions. I'm not familiar with SummarizedExperiment objects. Please try to [ask questions on StackOverflow](https://yihui.name/en/2017/08/so-gh-email/) instead (with minimal, self-contained, and reproducible examples). Thanks!
